v2/pkg/protocols/common/automaticscan/automaticscan.go

```diff
@@ -67,6 +67,8 @@ func New(opts Options) (*Service, error) {
 	if opts.ExecuterOpts.Options.Verbose {
 		gologger.Verbose().Msgf("Normalized mapping (%d): %v\n", len(mappingData), mappingData)
 	}
+	defaultTemplatesDirectories := []string{"nuclei-templates/"}
```

Comment:
We should try not to hard-code strings, nor store `/` or `\` characters in file names. The `filepath.Join()` method should take care of that.

Reply:
I'll modify the code to use the templates directory from config file which should solve this issue.